debian/rules

```diff
@@ -30,7 +30,7 @@ export DH_VIRTUALENV_INSTALL_ROOT = /usr/share
 override_dh_virtualenv:
 	dh_virtualenv \
 	  --extra-pip-arg "--no-cache-dir" \
-	  --python /usr/bin/python3 --preinstall=setuptools==40.3.0 --preinstall=pip==21.3.1 --preinstall=wheel --builtin-venv
+	  --python /usr/bin/python3 --preinstall=setuptools==52.0.0 --preinstall=pip==20.3.4 --preinstall=wheel==0.34.2 --builtin-venv
```

Comment:
You shouldn't downgrade `pip` here. Only `setuptools` needs to be upgraded for this to work on debian 11.

Reply:
You're right. It could also run even without pinned version at least for focal and bullseye. I'm affraid I did this to get it work for buster, but that's over the edge now.